.travis.yml

```diff
@@ -23,7 +24,7 @@ python:
 before_install:
   - if [ $CI_TARGET = tests ]; then
       eval "$(curl -Ss https://raw.githubusercontent.com/neovim/bot-ci/master/scripts/travis-setup.sh) nightly-x64";
-      pip install -q scrutinizer-ocular tox-travis;
```

Comment:
What is this change for?

Reply:
It is not really necessary, will remove it.